app/models/concerns/state.rb

```diff
@@ -15,4 +15,9 @@ module State
   CLOSED_STATE = 'closed'
   SELECTABLE_STATES = [OPEN_STATE, CLOSED_STATE, REJECTED_STATE, HIDDEN_STATE]
   SEARCHABLE_STATES = [OPEN_STATE, CLOSED_STATE, REJECTED_STATE]
+  TODO_LIST_STATES = [SPONSORED_STATE, VALIDATED_STATE, PENDING_STATE]
+  COLLECTING_SPONSORS_STATES = [VALIDATED_STATE, PENDING_STATE]
+  COLLECTING_SPONSORS = 'collecting sponsors'
```

Comment:
The ``COLLECTING_SPONSORS``, ``TODO_LIST``, and ``IN_MODERATION`` constants are only here to allow the ``.for_state`` scope to work.  We could be more careful about when we use ``for_state`` and prefer the ``in_moderation``, ``collecting_sponsors``, and ``todo_list`` scopes instead and that way we could drop these constants entirely.